problemsets-mat223/modules/module15.tex

```diff
@@ -89,8 +88,16 @@
 
 \begin{example}
 	Find the characteristic polynomial of $A=\mat{1&2\\3&4}$.
-
-	XXX Finish
+	
+	By the definition of Characteristic polynomial of A, we have: 
+	\begin{align*}
+	    \chr(A) &=\det(A-\lambda I)\\
```

Comment:
`\chr` -> `\Char`

Reply:
\Char will give a symbol like T(A) instead of char(A). Should I use \Char instead of \char?

Reply:
Never mind... I have no idea what happened to my computer... ...